WinUIGallery/ControlPages/FilePickerPage.xaml

```diff
@@ -0,0 +1,120 @@
+<!--
+//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+-->
+
+<Page x:Class="AppUIBasics.ControlPages.FilePickerPage"
+        xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:local="using:AppUIBasics"
+        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" mc:Ignorable="d">
+    <StackPanel>
+
+        <!-- 1) PICK A SINGLE FILE -->
+        <local:ControlExample x:Name="Example1" HeaderText="Pick a single file" 
+                              XamlSource="System/FilePickerSample1_xaml.txt"
+                              CSharpSource="System/FilePickerSample1_cs.txt">
+            <local:ControlExample.Example>
+                <StackPanel VerticalAlignment="Top">
+                    <TextBlock TextWrapping="Wrap" HorizontalAlignment="Left" Margin="0,0,0,15">
+                        Prompt the user to pick a single file.
+                    </TextBlock>
+                    <StackPanel VerticalAlignment="Top" Orientation="Horizontal">
+                        <Button x:Name="PickAFileButton" Content="Open a file"
+                                Click="PickAFileButton_Click" Margin="0,0,0,10"/>
+                        <TextBlock x:Name="OutputTextBlock1" TextWrapping="Wrap" Padding="20,5,0,0"/>
```

Comment:
Maybe, we can rename the TextBlocks to "PickAFileOutputTextBlock". "PickAPhotoOutputTextBlock"?

Reply:
I bolded the file names of all the examples (except the last one as it's different from the others + would make the code a lot longer). Let me know what you think!

Reply:
I also changed the names based on your suggestion. Thanks @chingucoding!

Reply:
I think with the bolding it looks really good, very good idea!